src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -134,12 +134,22 @@ private static FormattableString GetConversionFromFrameworkToRequestContent(this
         {
             if (TypeFactory.IsReadWriteDictionary(parameter.Type))
             {
-                return $"{typeof(RequestContentHelper)}.{nameof(RequestContentHelper.FromDictionary)}({parameter.Name})";
+                var conversionString = $"{typeof(RequestContentHelper)}.{nameof(RequestContentHelper.FromDictionary)}({parameter.Name})";
+                if (parameter.IsOptionalInSignature)
+                {
+                    conversionString = $"{parameter.Name} != null ? {conversionString} : null";
+                }
+                return $"{conversionString}";
```

Comment:
can we declare `conversionString` as `FormattableString` type, then it may not need to wrap again?